cypress/integration/models/developer/applicationinventory/analysis.ts

```diff
@@ -175,11 +175,21 @@ export class Analysis extends Application {
     }
 
     protected enableTransactionAnalysis() {
-        cy.get(enableTransactionAnalysis).click();
+        cy.get(enableTransactionAnalysis)
+            .invoke("is", ":checked")
```

Comment:
Do we need this part ? For analysis all the checkboxes are unchecked by default and it does not save previous analysis' data .

Reply:
This is not required, but I'm just being cautious . 
![application_analysis](https://user-images.githubusercontent.com/5183021/230685829-b0385fca-a065-4eee-bea9-54ab2cd39b63.png)